src/modules/organization/home/team/org-event-list-card/OrgEventListCard.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div class="flex w-full shadow-sm rounded-2xl overflow-hidden bg-white">
+    <section class="w-16.5 flex-shrink-0 min-h-full">
+      <LazyImage
+        :width="150"
+        :height="150"
+        alt="will change to api"
+        :url="event.img"
+        :placeholder="event.imgHash"
+        class="object-cover w-full h-full"
+      />
+    </section>
+    <section class="relative py-2 pl-2 pr-8 w-full">
+      <h1 class="text-blue-10 font-heading text-2xl mb-2">{{ event.title }}</h1>
+      <div class="flex flex-col lg:flex-row lg:justify-between">
+        <div class="flex items-center mb-2 lg:mb-0">
+          <section class="flex items-center mr-2">
+            <base-icon
+              :width="14"
+              :height="14"
+              class="text-primary mr-1 flex-shrink-0"
+              ><CalendarIcon
+            /></base-icon>
+            <p>{{ event.date }}</p>
+          </section>
+          <section class="flex items-center mr-2">
+            <base-icon

```

Comment:
```
            <base-icon
              :width="14"
              :height="14"
              class="text-primary mr-1 flex-shrink-0"
              ><ClockIcon
            /></base-icon>
            <p>{{ event.time }}</p>
```
This should be treated as a new component, because it has many usage
https://www.figma.com/file/Dwjrkq9zwlhJxehVHgqrIg/UI-Main?node-id=953%3A2441
https://www.figma.com/file/Dwjrkq9zwlhJxehVHgqrIg/UI-Main?node-id=1486%3A0
https://www.figma.com/file/Dwjrkq9zwlhJxehVHgqrIg/UI-Main?node-id=1562%3A4545

Reply:
Can you elaborate? What is this? Using base icon?

Reply:
![image](https://user-images.githubusercontent.com/12471844/114278098-4461ed00-9a58-11eb-8c26-1986e48e07f9.png)